scan-payment/src/main/java/com/getbouncer/scan/payment/analyzer/PaymentCardOcrAnalyzer.kt

```diff
@@ -0,0 +1,67 @@
+package com.getbouncer.scan.payment.analyzer
+
+import android.util.Log
+import com.getbouncer.scan.framework.Analyzer
+import com.getbouncer.scan.framework.AnalyzerFactory
+import com.getbouncer.scan.payment.ml.*
+import com.getbouncer.scan.payment.ml.ssd.DetectionBox
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.async
+
+
+data class PaymentCardPanOcrAnalyzerOutput(
```

Comment:
nit: make this an inner class of `PaymentCardOcrAnalyzer` and name it something like `Prediction` or `Output`